sound/soc/sof/ipc4-priv.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+#ifndef __SOUND_SOC_SOF_IPC4_PRIV_H
+#define __SOUND_SOC_SOF_IPC4_PRIV_H
+
+#include "sof-priv.h"
+
+/**
+ * struct sof_ipc4_private_data - IPC4-specific data
+ * @num_fw_modules : Number of modules in base FW
+ * @fw_modules: Array of base FW modules
+ * @manifest_fw_hdr_offset: FW header offset in the manifest
+ */
+struct sof_ipc4_data {
```

Comment:
the name doesn't match the kernel-doc, and it's as vague as can be. 'data' could refer to just about anything.

sof_ipc4_fw_data?

Reply:
Oops, I have overlooked this.

Yes, `sof_ipc4_fw_data` is more appropriate